mets.md

```diff
@@ -30,6 +30,10 @@ $> exiftool output.tif |grep 'X Resolution'
 "150"
 ```
 
+However, since technical metadata about pixel density is so often lost in
+conversion or inaccurate, processors should assume **300 ppi** for images with
```

Comment:
@kba I thought that we agreed that I would have the chance to reformulate that a bit...